app/assets/javascripts/dynamic_search.js

```diff
@@ -0,0 +1,164 @@
+/**
```

Comment:
We don't currently test the autocomplete search, but with this rollout it's probably a good idea. Ananyo and I put together a Jasmine-based test framework for JavaScript code you could use for a basic test of "typing in the box makes it show", following the format of existing tests here: https://github.com/publiclab/plots2/blob/master/spec/javascripts/plots2_spec.js

HTML fixtures (test markup templates) to work with are here: https://github.com/publiclab/plots2/tree/master/spec/javascripts/fixtures